Source/WebKit/StorageProcess/StorageProcess.cpp

```diff
@@ -310,11 +310,13 @@ void StorageProcess::fetchWebsiteData(PAL::SessionID sessionID, OptionSet<Websit
 {
     auto websiteData = std::make_unique<WebsiteData>();
     WebsiteData* rawWebsiteData = websiteData.get();
+    UNUSED_PARAM(rawWebsiteData);
```

Comment:
remove rawWebsiteData

Reply:
can't remove it - it's used in #if ENABLE(SERVICE_WORKER) or #if ENABLE(INDEXED_DATABASE) blocks, see below